kms/spec/quickstart_spec.rb

```diff
@@ -0,0 +1,63 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/apis/cloudkms_v1beta1"
+
+describe "Key Management Service Quickstart" do
+  Cloudkms = Google::Apis::CloudkmsV1beta1
+
+  def create_test_key_ring parent, key_ring_id
+    client = Cloudkms::CloudKMSService.new
+    client.authorization = Google::Auth.get_application_default(
+      %[https://www.googleapis.com/auth/cloud-platform]
+    )
+
+    client.create_project_location_key_ring parent, Cloudkms::KeyRing.new,
+                                            key_ring_id: key_ring_id
+  end
+
+  before :all do
+    # Note: Code samples set constants, and constants cannot be set inside
```

Comment:
Setting constants in method definitions isn't why we set `VERBOSE = nil`

The method definitions are handled by the lambdas.

Because the samples all define a `Cloudkms` constant, `$VERBOSE` is disabled to silence "already initialized constant" warnings.